protypes/Eclipse GUI with Wizards and Dialogs/src/Timeout.java

```diff
@@ -0,0 +1,225 @@
+import org.eclipse.jface.wizard.WizardPage;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.KeyEvent;
+import org.eclipse.swt.events.KeyListener;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.events.SelectionListener;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Text;
+
+/**
+ * Page 2 of the wizard.
+ * 
+ * @author Christoph Michelbach
+ *
+ */
+public class Timeout extends WizardPage {
+	/**
+	 * The title of this page.
+	 */
+	private static String title = "Timeout";
+
+	/**
+	 * The description of this page.
+	 */
+	private static String description = "Blah blah ... a timeout can be used.";
+
+	/**
+	 * Numeric value for "adaptive timeout".
+	 */
+	private final int adaptiveTimout = -2;
+
+	/**
+	 * Numeric value for "no timeout".
+	 */
+	private final int noTimeout = -1;
+
+	/**
+	 * A textbox for the timeout in seconds (if the timeout is set manually).
+	 */
+	private Text tbTimeoutSeconds;
```

Comment:
I like the abbreviations because I think that they make it more readable and are universally applicable for GUIs, but okay. I'll change it.